src/solutions/optimization_solutions.jl

```diff
@@ -64,3 +64,7 @@ function Base.summary(io::IO, A::AbstractOptimizationSolution)
           type_color, eltype(A.u),
           no_color)
 end
+
+function observed(A::AbstractOptimizationSolution, sym)
+    getobserved(A)(sym, A.u, A.cache.p)
+end
```

Comment:
Does this not just work by the abstractnotimesolution fallbacks already? I don't think this is correct.

Reply:
This?
https://github.com/SciML/SciMLBase.jl/blob/3a2c5ed100f7dc76ecce8233e2e377ac917fd158/src/solutions/solution_interface.jl#L137

Reply:
oh it's a prob vs cache thing? Can we put this there so it's more clear?